cms/static/js/views/paging_header.js

```diff
@@ -19,83 +25,99 @@ define(["underscore", "backbone", "gettext", "text!templates/paging-header.under
                     } else {
                         // Translators: sample result:
                         // "Showing 0-9 out of 25 total, sorted by Date Added descending"
-                        message = gettext('Showing %(current_item_range)s out of %(total_items_count)s, sorted by %(sort_name)s descending');
+                        message = gettext('Showing {currentItemRange} out of {totalItemsCount}, sorted by {sortName} descending');
                     }
                 }
 
-                return '<p>' + interpolate(message, {
-                        current_item_range: this.currentItemRangeLabel(),
-                        total_items_count: this.totalItemsCountLabel(),
-                        asset_type: asset_type,
-                        sort_name: this.sortNameLabel()
-                    }, true) + "</p>";
+                return HtmlUtils.interpolateHtml(message, {
+                    currentItemRange: this.currentItemRangeLabel(),
+                    totalItemsCount: this.totalItemsCountLabel(),
+                    assetType: assetType,
+                    sortName: this.sortNameLabel()
+                });
             },
 
             currentItemRangeLabel: function() {
                 var view = this.view,
                     collection = view.collection,
-                    start = collection.start,
+                    start = (collection.getPageNumber() - 1) * collection.getPageSize(),
                     count = collection.size(),
-                    end = start + count;
-                return interpolate('<span class="count-current-shown">%(start)s-%(end)s</span>', {
+                    end = start + count,
+                    htmlMessage = HtmlUtils.HTML('<span class="count-current-shown">{start}-{end}</span>');
+
+                return HtmlUtils.interpolateHtml(htmlMessage, {
                     start: Math.min(start + 1, end),
                     end: end
-                }, true);
+                });
             },
 
             totalItemsCountLabel: function() {
-                var totalItemsLabel;
+                var totalItemsLabel,
+                    htmlMessage = HtmlUtils.HTML('<span class="count-total">{totalItemsLabel}</span>');
+
                 // Translators: turns into "25 total" to be used in other sentences, e.g. "Showing 0-9 out of 25 total".
-                totalItemsLabel = interpolate(gettext('%(total_items)s total'), {
-                    total_items: this.view.collection.totalCount
-                }, true);
-                return interpolate('<span class="count-total">%(total_items_label)s</span>', {
-                    total_items_label: totalItemsLabel
+                totalItemsLabel = HtmlUtils.interpolateHtml(gettext('{totalItems} total'), {
+                    totalItems: this.view.collection.getTotalRecords()
                 }, true);
```

Comment:
This function doesn't take the third boolean parameter so you can just remove the ``true``.